src/scss/components/_button.scss

```diff
@@ -16,69 +16,63 @@
 **/
 
 /**
-  * @styles buttons_mc Medium Contrast
```

Comment:
If we'll remove this documentation, let's remove the button too. 

~Break changes is coming~